Makefile

```diff
@@ -1,12 +1,12 @@
 testjs: ## Clean and Make js tests
 	yarn test
 
-testpy: ## Clean and Make unit tests
-	python3.7 -m pytest -v jupyterfs/tests --cov=jupyterfs
+testpy: ## Clean and Make py tests
```

Comment:
Good call with branch coverage, can you add this to nbcelltests if its not already there (@ceball)

Reply:
It's already there 

(I'm not able to mark this as resolved)